src/lambdas/processDataRequest/checkDataTransferStatus.ts

```diff
@@ -72,7 +72,7 @@ export const checkDataTransferStatus = async (zendeskId: string) => {
     !glacierRestoreStillInProgress
   ) {
     console.log(
-      'Restore/copy process complete. Placing zendeskId on InitiateAthenaQueryQueue'
+      `Restore/copy process complete. Placing zendeskId '${zendeskId}' on InitiateAthenaQueryQueue`
```

Comment:
I see you are triggering the athena queue from within the application code. I had thought we would have a new spec file under integration tests where this would be done, and somehow point the lambda to the new integration test audit bucket?

Reply:
We could do, this just adds a check that the copy process is complete for that ticket id. The fact it goes to Athena isn't checked beyond this point.